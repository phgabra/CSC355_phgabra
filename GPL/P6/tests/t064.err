Semantic error on line 5: Variable 'a' previously declared.
Semantic error on line 6: Variable 'a' previously declared.
Semantic error on line 9: Variable 'b' previously declared.
Semantic error on line 10: Variable 'b' previously declared.
Semantic error on line 13: Variable 'c' previously declared.
Semantic error on line 14: Variable 'c' previously declared.
Semantic error on line 17: Variable 'd' previously declared.
Semantic error on line 18: Variable 'd' previously declared.
Semantic error on line 21: Variable 'f' previously declared.
Semantic error on line 22: Variable 'f' previously declared.
