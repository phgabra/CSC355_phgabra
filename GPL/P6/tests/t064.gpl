// should issue many "previously declared" errors


int a;
double a;
string a;

int b[10];
double b[10];
string b[10];

string c[10];
int c;
double c;

string d;
int d;
double d[10];

double f;
int f[100];
string f;

