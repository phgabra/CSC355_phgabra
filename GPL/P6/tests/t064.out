gpl.cpp::main()
  input file(tests/t064.gpl)
  random seed(42)
  read_keypresses_from_standard_input(false)
  dump_pixels(false)
  symbol_table(true)
  graphics(true)

gpl.cpp::main() Calling yyparse()


gpl.cpp::main() after call to yyparse().

10 errors found.
gpl giving up.
